### DISCUSSION TITLE
[Question] Can I use neoconf without lsp?

### DISCUSSION THREAD

**Discussion by qlibp** [folke/neoconf.nvim#72] [Q&A]
Neoconf seems to set itself as a configuration manager of lsp server. Therefore, it requires lspconfig plugin. 

But is it possible to use it as a general configuration manager for non-lsp plugins?

If we target it as a general configuration manager(manager for both lsp&non-lsp plugins), then it's kind of weird that neoconf will crash without installing lspconfig first.

[a comment by an automated account is not shown]

**Comment by nerdwave-nick** [comment]
I'd probably like to bump this. Especially now with nvim 0.11 lsp upgrades. I've removed lspconfig from my config, but I can't use neoconf anymore due to it crashing without lspconfig. And that despite having disabled all lspconfig/lsp specific config options as well. I think this plugin could really benefit from not being tied to lspconfig
